packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -2519,7 +2519,7 @@ void main() {
     ));
 
     semantics.dispose();
-  });
+  }, skip: true);
```

Comment:
I think this test can be fixed by using a pumpAndSettle above instead of pump at line 2490.

```diff
- await tester.pump();
+ await tester.pumpAndSettle();
```

Reply:
The golden test is also failing for a similar reason.  With the change in this PR, the clipboard status becomes `pastable` right away, whereas previously it had to wait for an asynchronous call before updating.  Changing it to `pumpAndSettle` will allow the cursor to be shown, but it will also still break the test because the field underline has a chance to fade in.  I think the best thing to do is to change to `pumpAndSettle` and then accept the new image.